fury/io.py

```diff
@@ -281,3 +282,51 @@ def save_polydata(polydata, file_name, binary=False, color_array_name=None):
         writer.SetFileTypeToBinary()
     writer.Update()
     writer.Write()
+
+
+def load_sprite_sheet(sheet_path, nb_rows, nb_cols, as_vtktype=False):
+    """Process and load sprites from a sprite sheet
+
+    Parameters
+    ----------
+    sheet_path: str
+        Path to the sprite sheet
+    nb_rows: int
+        Number of rows in the sprite sheet
+    nb_cols: int
+        Number of columns in the sprite sheet
+    as_vtktype: bool, optional
+        If True, the output is a vtkImageData
+
+    Returns
+    -------
+    Dict containing the processed sprites.
+    """
+    sprite_dicts = {}
+    sprite_sheet = Image.fromarray(load_image(sheet_path))
```

Comment:
I do not think you need `Image.fromarray`. The NumPy array should be enough for what you need to do. 
it is also easy to crop a NumPy array via slicing or to get the width or height.